<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsValueMapWidget</class>
 <widget class="QWidget" name="QgsValueMapWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>701</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="valueMapLabel">
     <property name="text">
      <string>Combo box with predefined items. Value is stored in the attribute, description is shown in the combo box.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="loadFromLayerButton">
     <property name="text">
      <string>Load Data from Layer</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="loadFromCSVButton">
     <property name="text">
      <string>Load Data from CSV File</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>339</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" colspan="3">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTableWidget" name="tableWidget">
       <column>
        <property name="text">
         <string>Value</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Description</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mValueMapErrorsLabel">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>100</height>
        </size>
       </property>
       <property name="text">
        <string>No errors</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="addNullButton">
       <property name="text">
        <string>Add &quot;NULL&quot; value</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="removeSelectedButton">
       <property name="text">
        <string>Remove Selected</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>518</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>loadFromLayerButton</tabstop>
  <tabstop>loadFromCSVButton</tabstop>
  <tabstop>tableWidget</tabstop>
  <tabstop>addNullButton</tabstop>
  <tabstop>removeSelectedButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
